system/index.js

```diff
@@ -88,7 +88,9 @@ module.exports = async (message) => {
     throw error;
   }
 
-  return result;
+  if (typeof result !== 'undefined') {
+    return result
+  }
```

Comment:
What's the return value if we don't `return result;` here?

Reply:
I'm about to test this out. Thinking about it as things build I don't think this will change anything. This might be an upstream riff bug that is handling undefined like this. You can't _not_ return something really.